core-services/prow/02_config/OWNERS

```diff
@@ -1,5 +1,6 @@
 approvers:
   - dptp
+  # The following approvers are just for generate-boskos.py and _boskos.yaml:
```

Comment:
Can we do it like this?

```yaml
- wking # Just for generate-boskos.py and _boskos.yaml
```

Hopefully, we do not have a formatter to prevent it.

Reply:
Done with aa7ac65108 -> 534bc9fab1